a11y.js

```diff
@@ -67,6 +67,7 @@ const pageTypes = {
     '/iframe', // same issues as in mediaEmbedErrorsToSuppress but the DOM path is different
     "//div[@id='root']/div/div[1]/main/div[37]/div/div/div", // issue with IDT2 includes
     "//div[@id='ns_datapic_royal-engagements']",
+    "//div[@id='ns_datapic_env-test-2']",
```

Comment:
What does this addition do, @karinathomasbbc?

Reply:
Makes the a11y tests (for STYs) pass locally.